metaflow/datastore/datastore.py

```diff
@@ -4,7 +4,10 @@
 import time
 from hashlib import sha1
 from functools import partial
-from collections import Sequence
+try:
```

Comment:
Actually, Sequence does not seem to be used anywhere in this file so maybe we can just remove the line and no add anything :).

Reply:
Thanks, I just did a find and replace. Removed the unused import now.